pkg/service/connect_service.go

```diff
@@ -245,170 +245,103 @@ func (s *ConnectService) Notify(n Notification) {
 	}
 }
 
-func (s *ConnectService) ResolveBoolean(
-	ctx context.Context,
-	req *connect.Request[schemaV1.ResolveBooleanRequest],
-) (*connect.Response[schemaV1.ResolveBooleanResponse], error) {
+type response[T constraints] interface {
+	SetResult(value T, variant, reason string) error
+}
+
+type constraints interface {
+	bool | string | map[string]any | float64 | int64
+}
```

Comment:
Not a must but in my view these interfaces fits best inside `connect_service_types.go` :)

Reply:
Good suggestion, done :)